### PR TITLE
[ImagingUpload.pm] Look inside file type and not file path for the word dicom

### PR DESCRIPTION
**Pull request by gluneau**
Previous file_type content:
/tmp/ImagingUpload-12-48-Rv9qJY/LEGO_PHANTOM_HOS_20160830/DICOM/.test: ASCII text

New file_type content:
DICOM medical imaging data

### CODE REVIEW

File: uploadNeuroDB/NeuroDB/ImagingUpload.pm [hunk 1]
```diff
@@ -417,7 +417,8 @@ sub isDicom {
     my $this         = shift;
     my ($dicom_file) = @_;
     my $cmd    = "file $dicom_file";
-    my $file_type    = `$cmd`;
+    my $cmd_log = `$cmd`;
+    my ($file_path, $file_type) = split /:/, $cmd_log;
```
Comment (MounaSafiHarab) [inline uploadNeuroDB/NeuroDB/ImagingUpload.pm]: match on *last* DICOM medical imaging data

Comment (driusan) [inline uploadNeuroDB/NeuroDB/ImagingUpload.pm]: `DICOM medical imaging data$`

**Comment by MounaSafiHarab** [comment]
@gluneau 

works; scripts fail now at the isDicom check step as opposed to the PatientName extraction step (see messages I get below; although I feel that either way the script would fail with Okay messages). 

but can you please change the description above? a .DS_Store file would not go through this check in the first place (check line 190). I am guessing you tested this before merging the pull request that excludes the .DS_Store from the check.

With my tests, here is what I get withOUT your changes:
`E: DcmElement: CommandField (0000,0100) larger (828667202) than remaining bytes in file
E: dcmdump: I/O suspension or premature end of stream: reading file: /data/not_backed_up/ImagingUpload-12-38-aAfrn0/DICOM/test/.test
Spool message is: 
The patient name cannot be extracted `

and WITH your change
`/data/not_backed_up/ImagingUpload-12-36-SfJ97X/DICOM/test/.test is not of type DICOM 
Spool message is: 
ERROR: There are 1 file(s) which are not of type DICOM `

**Comment by MounaSafiHarab** [comment]
try `file -b` as per @driusan

**Pull request merged**